tools/lumpy/lumpy.xml

```diff
@@ -0,0 +1,261 @@
+<tool id="lumpy" name="lumpy-sv" version="0.2">
+    <description>find structural variants</description>
+    <requirements>
+        <requirement type="package" version="0.2.12">lumpy-sv</requirement>
+        <requirement type="package" version="1.3.1">samtools</requirement>
+        <requirement type="package" version="1.11.2">numpy</requirement>
```

Comment:
Since the wrapperscript doesn't look like it's python3 compatible I would fix the numpy version to py27.
I think `1.11.2=py27` in the version tag should do it.